terraform/install.sh

```diff
@@ -201,12 +202,31 @@ applyTerraform() {
     terraform init -backend-config "bucket=${bucket_name}" -lockfile=false # lock-free to prevent access fail
   fi
 
-  log "Apply Terraform automation"
+  #build Terraform apply command 
+  terraform_command="terraform apply -auto-approve -var=\"project_id=${project_id}\" -var=\"bucket_name=${bucket_name}\" -var=\"skip_loadgen=${skip_loadgen:-false}\""
+
+  #If billing account provided specify it 
   if [[ -n "$billing_id" ]]; then
-    terraform apply -auto-approve -var="billing_account=${billing_acct}" -var="project_id=${project_id}" -var="bucket_name=${bucket_name}" -var="skip_loadgen=${skip_loadgen:-false}"
-  else
-    terraform apply -auto-approve -var="project_id=${project_id}" -var="bucket_name=${bucket_name}"  -var="skip_loadgen=${skip_loadgen:-false}"
+    terraform_command+=" -var=\"billing_account=${billing_acct}\"" 
   fi
+  #Check application version
+  #If cluster already exist leave original version
+  gke_location="$(gcloud container clusters list --format="value(location)" --filter name=cloud-ops-sandbox)"
+  if [[ -n "$gke_location" ]]; then 
+    gke_version="$(gcloud container clusters describe cloud-ops-sandbox --region "${gke_location}"  --format="value(resourceLabels.version)")"
```

Comment:
do we deploy the cluster into region? does the command work if the `--region` parameter is passed with the value of the zone?

Reply:
Changed to zone